examples/my_kentik_portal_example_test.go

```diff
@@ -0,0 +1,49 @@
+//+build examples
+
+package examples
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+func TestMyKentikPortalAPIExample(t *testing.T) {
+	defer func() {
+		if err := recover(); err != nil {
+			t.Fatal(err)
+		}
+	}()
+	runCRUDexample()
+	getAllTenants()
+}
+
+func runCRUDexample() {
+	client := NewClient()
+
+	fmt.Println("### GET")
+	tenant, err := client.MyKentikPortal.Get(context.Background(), 577)
+	PanicOnError(err)
```

Comment:
today's morning with Daniel we agreed on not panicking but returning errors which is more verbose but also more GOthic :)